crates/rome_js_formatter/src/js/lists/call_argument_list.rs

```diff
@@ -1,13 +1,23 @@
 use crate::generated::FormatJsCallArgumentList;
 use crate::prelude::*;
+use crate::utils::fmt_arguments_multi_line;
+use rome_formatter::write;
 use rome_js_syntax::JsCallArgumentList;
 
 impl FormatRule<JsCallArgumentList> for FormatJsCallArgumentList {
     type Context = JsFormatContext;
 
     fn fmt(node: &JsCallArgumentList, f: &mut JsFormatter) -> FormatResult<()> {
-        f.join_with(&soft_line_break_or_space())
-            .entries(node.format_separated(token(",")))
-            .finish()
+        if node.len() == 0 {
+            return write!(f, [empty_element()]);
```

Comment:
Nit: You don't have to perform a write. You can return `Ok(())` in cases where you don't want to write any output
```suggestion
            return Ok(());
```